index.html

```diff
@@ -466,10 +466,10 @@
       automatically generated by the application server or an optimization
       proxy (e.g. a CDN).</p>
       <pre class="example">
-  Link: &lt;https://example.com/font.woff&gt;; rel=preload; as=font
+  Link: &lt;https://example.com/font.woff2&gt;; rel=preload; as=font; type='font/woff2'
   Link: &lt;https://example.com/app/script.js&gt;; rel=preload; as=script
   Link: &lt;https://example.com/logo-hires.jpg&gt;; rel=preload; as=image
-  Link: &lt;https://fonts.example.com/font.woff&gt;; rel=preload; as=font; crossorigin
+  Link: &lt;https://fonts.example.com/font.woff2&gt;; rel=preload; as=font; crossorigin; type='font/woff2'
```

Comment:
@yoavweiss: Yes, I'll do that, but let me explain the original reason first.

I'm using `font/woff2` as value for `type`:
```
Link: </path/to/font.woff2>; rel=preload; as=font; type='font/woff2'
```

The entire `Link` header looks like this (paths are shortened):
```
<main.css>; rel=preload; as=style, <main.js>; rel=preload; as=script, <light.woff2>; rel=preload; as=font; type='font/woff2', <regular.woff2>; rel=preload; as=font; type='font/woff2', <semibold.woff2>; rel=preload; as=font; type='font/woff2'
```

I tried adding a `type` value for fonts, because I noticed that the 3 fonts are pushed fine on the first (cold) page request. However any following requests the 3 fonts are downloaded twice and Google Chrome show this warning 3 times:

```
The resource {...}.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it wasn't preloaded for nothing.
```

I thought maybe adding a `type` would solve that issue. The site is behind Cloudflare and you can test it yourself: https://laylamartin.com/contact/